aws/rust-runtime/aws-runtime/src/user_agent/metrics.rs

```diff
@@ -141,6 +151,23 @@ impl ProvideBusinessMetric for SmithySdkFeature {
             Paginator => Some(BusinessMetric::Paginator),
             GzipRequestCompression => Some(BusinessMetric::GzipRequestCompression),
             ProtocolRpcV2Cbor => Some(BusinessMetric::ProtocolRpcV2Cbor),
+            FlexibleChecksumsReqCrc32 => Some(BusinessMetric::FlexibleChecksumsReqCrc32),
```

Comment:
That could be helpful, but I am torn. It would certainly eliminate boilerplate, but without most dev tools having some kind of easy macro expansion utility I think it also hides some of the details. Given that it is only really used for UA things at the moment I would leave it as is, but if we were using this pattern in multiple places a macro would likely be worth it.

Reply:
Yeah, that sounds good.